posts/ja/2022-11-01-letstry-instant-on-intel-mac-en.adoc

```diff
@@ -0,0 +1,280 @@
+---
+layout: post
+title: Let's try Open Liberty InstantOn on Intel Mac
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/takakiyo
+author_github: https://github.com/takakiyo
+seo-title: Let's try Open Liberty InstantOn on Intel Mac - OpenLiberty.io
+seo-description: Let's try Open Liberty InstantOn on Intel Mac
+blog_description: "Let's try Open Liberty InstantOn on Intel Mac"
+//open-graph-image: https://openliberty.io/img/blog/instantonperf.png
+//blog-available-in-languages: ['ja']
+additional_authors: 
+- name: Hiroko Takamiya
+  github: https://github.com/una-tapa
+  image: https://avatars0.githubusercontent.com/una-tapa
+---
+= Let's try Open Liberty InstantOn on Intel Mac
+Takakiyo Tanaka <https://github.com/takakiyo>
+:imagesdir: /
+:url-prefix:
+:url-about: /
+
+== What is Liberty InstantOn ? 
+
+Liberty InstantOn was announced on September 29, 2022. It is a technology that launches a full-featured Java EE application server in hundreds of milliseconds.
```

Comment:
```suggestion
Liberty InstantOn was announced on September 29, 2022. It is a technology that launches a full-featured cloud-native Java runtime supporting MicroProfile and Jakarta EE APIs in hundreds of milliseconds.
```

Reply:
Line 26 will be updated with Tom's suggestion
"It is a technology that provides a full-featured cloud-native Java runtime supporting MicroProfile and Jakarta WebProfile APIs that enables applications to start in hundreds of milliseconds."